templates/rsyslog.conf.erb

```diff
@@ -87,7 +87,9 @@ $ActionQueueSaveOnShutdown on # save messages to disk on shutdown
 $ActionQueueType LinkedList   # run asynchronously
 $ActionResumeRetryCount -1    # infinite retries if host is down
 <% end -%>
-<%= @source_facilities %> <% if @transport_protocol == 'tcp' -%>@<% end -%>@<%= @log_server -%>:<%= @log_server_port %>
+<% for server in @log_server -%>
```

Comment:
Instead of `for`, use `.each`. Also need to use `.class` with a conditional so that if the class is a string, we do what was already there and if it is an array then we do the iteration. You can't use .each (or your for loop) with a string in ruby > 1.8.7, which explain the Travis-ci results.